packages/vc.js/src/__tests__/waci-pex/__fixtures__/credential-create.json

```diff
@@ -0,0 +1,31 @@
+{
+  "items": [
+    {
+      "@context": [
+        "https://www.w3.org/2018/credentials/v1",
+        "https://w3id.org/security/suites/bls12381-2020/v1",
+        "https://w3id.org/vaccination/v1"
+      ],
+      "id": "urn:uvci:af5vshde843jf831j128fj",
+      "type": ["VaccinationCertificate", "VerifiableCredential"],
+      "description": "COVID-19 Vaccination Certificate",
+      "name": "COVID-19 Vaccination Certificate",
+      "credentialSubject": {
+        "id": "urn:bnid:_:c14n2",
```

Comment:
this looks like the problem to me since it wasn't in the original vc

Reply:
This should be present in the derivation IF the original VC did not contain a `credentialSubject.id`...